### DISCUSSION TITLE
Feature request - Status screen when playing seq directly from Xlights - DDP source

### DISCUSSION THREAD

**Discussion by brmeadows2841** [forkineye/ESPixelStick#483] [Ideas]
**Is your feature request related to a problem? Please describe.**

It would be good if when playing a seq directly from Xlights if on the home/status page the name of the seq or some indication of what is being displayed.

**Describe the solution you'd like**

It would be good if when playing a seq directly from Xlights if on the home/status page the name of the seq or some indication of what is being displayed.
![ESpixelstick screen shot of status screen](https://user-images.githubusercontent.com/81775646/154551502-c757109d-32d8-49c0-98a8-4f8508ba1f92.png)

**Additional context**

**Comment by MartinMueller2003** [comment]
The problem is that the name of the sequence is not in the DDP data stream. If xLights were to add that and let us know the protocol for extracting the info, I would be happy to add this.

**Comment by brmeadows2841** [comment]
Im guessing this is not doable as part of the DDP protocol.
I got this from Keith Wesley of Xlights.

[keithsw1111](https://github.com/keithsw1111) commented [yesterday](https://github.com/smeighan/xLights/issues/3134#issuecomment-1049712106)

In DDP it is not possible … there is no place in the protocol to include it. E131 does have a spot to say where data is coming from (64 chars from memory) but right now we just send xLights. We don’t send the sequence name.

**Comment by brmeadows2841** [comment]
The feature may not be in the DDP stream but apparently its there somewhere, when an ESP32 CAM is set as FFP remote.

<html>
<body>
<legend class="esps-legend" style="box-sizing: border-box; border-width: 0px 0px 1px; border-top-style: initial; border-right-style: initial; border-bottom-style: solid; border-left-style: initial; border-top-color: initial; border-right-color: initial; border-bottom-color: rgb(229, 229, 229); border-left-color: initial; border-image: initial; padding: 0px; display: block; width: 554.994px; margin-bottom: 5px; font-size: 21px; line-height: inherit; color: rgb(51, 51, 51); margin-top: 5px; font-family: &quot;Helvetica Neue&quot;, Helvetica, Arial, sans-serif; font-style: normal; font-variant-ligatures: normal; font-variant-caps: normal; font-weight: 400; letter-spacing: normal; orphans: 2; text-align: start; text-indent: 0px; text-transform: none; white-space: normal; widows: 2; word-spacing: 0px; -webkit-text-stroke-width: 0px; background-color: rgb(255, 255, 255); text-decoration-thickness: initial; text-decoration-style: initial; text-decoration-color: initial;">FPP Remote Status</legend>

Total Sync Packets: | 133651
-- | --
Sync Adjustments: | 40
Total Files Played: | 4
Master IP: | 10.0.0.10
Filename: | A Mad Russians Christmas-TSO LOR_1 changed CandyCaneLayout.fseq
Time Elapsed: | 00:00
Time Remaining: | 00:00
Last Error: | ParseFseqFile:: Could not open file: filename: 'A Mad Russians Christmas-TSO LOR_1 changed CandyCaneLayout.fseq'


</body>
</html>

**Comment by MartinMueller2003** [comment]
FPP fseq sync messages carry a lot of information. In general, if some data point is available, O added it to the display.
